CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ if(CORECLR_SET_RPATH)
 endif(CORECLR_SET_RPATH)
 
 OPTION(CMAKE_ENABLE_CODE_COVERAGE "Enable code coverage" OFF)
+OPTION(CMAKE_WARNINGS_ARE_ERRORS "Warnings are errors" ON)
```

Comment:
Can you please name the option CLR_CMAKE_WARNINGS_ARE_ERRORS instead? Besides that option above that I've not noticed during past PRs, we don't use options starting with CMAKE_, since that is reserved for variables of the cmake itself. And if you could also rename the CMAKE_ENABLE_CODE_COVERAGE, it would be awesome.